examples/fixture/asm/kimchi/public_output_bool.asm

```diff
@@ -0,0 +1,29 @@
+@ noname.0.7.0
+
+DoubleGeneric<1>
+DoubleGeneric<1>
+DoubleGeneric<1,1,-1>
+DoubleGeneric<1,0,0,0,-2>
+DoubleGeneric<1,0,-1,0,6>
+DoubleGeneric<1,0,0,0,-8>
+DoubleGeneric<1,1>
+DoubleGeneric<1,1,-1>
+DoubleGeneric<0,0,-1,1>
+DoubleGeneric<1,1>
+DoubleGeneric<1,0,-1,0,1>
+DoubleGeneric<1,-1>
+DoubleGeneric<0,0,-1,1>
+DoubleGeneric<1>
+DoubleGeneric<1,-1>
+(0,0) -> (14,0)
+(1,0) -> (2,1)
+(2,2) -> (3,0) -> (4,0)
+(4,2) -> (6,0)
+(5,0) -> (7,0)
+(6,1) -> (7,1)
+(7,2) -> (8,1) -> (12,1)
+(8,2) -> (11,0)
+(9,0) -> (12,0) -> (14,1)
```

Comment:
I wish we got better debugging for gadgets like the `==`. [Maybe I'll create an issue for that](https://github.com/zksecurity/noname/issues/148)